tests/patterns/python/pip614-extended-decorator-grammer1.py

```diff
@@ -0,0 +1,15 @@
+#ERROR:
```

Comment:
if the fix address just a parsing issue with a language, there is no need to add a .sgrep example for it.
Instead you can just add the new test file in tests/parsing/python/

If you also want to make sure that we accept the new syntax in the pattern, then yes you want a .sgrep file.

Reply:
Understood: I was worried if the choice of translation between Python AST ==> Generic AST would ruin attribute matching, and was why I included these patterns test.

it did catch a matching bug for when I chose to translate all decorators as `G.OtherAttribute`, instead of attempting to trans some as `G.NamedAttr`.